tests/integration/test_core.py

```diff
@@ -92,31 +93,6 @@ async def test_mysql_relation(
     )
 
 
-@pytest.mark.asyncio
```

Comment:
Should this test be removed? Is it no longer necessary?

Reply:
This test case is move to a different location in the same file.

Reply:
Ah